spec/typescript/exceljs.spec.ts

```diff
@@ -24,17 +27,21 @@ describe('typescript', () => {
     ws.getCell('A1').value = 7;
 
     const wb2 = new ExcelJS.Workbook();
-    const stream = wb2.xlsx.createInputStream();

```

Comment:
This test was previously failing due to removal of `createInputStream`, so I've overwritten the existing test case rather than adding a new one.